operational/bill_processing_errors_notification/bill_processing_errors_notification.pt

```diff
@@ -0,0 +1,179 @@
+name "Bill Processing Error Notification"
+rs_pt_ver 20180301
+type "policy"
+short_description "Collects all currently applied policies and raises an incident for any in an error state. See the [README](https://github.com/flexera/policy_templates/tree/master/operational/bill_processing_errors_notification/) and [docs.flexera.com/flexera/EN/Automation](https://docs.flexera.com/flexera/EN/Automation/AutomationGS.htm) to learn more."
+long_description ""
+severity "high"
+category "Operational"
+default_frequency "daily"
+info(
+    version: "1.0",
+    provider: "Flexera Cloud Management",
+    service: "",
+    policy_set: ""
+  )
+
+###############################################################################
+# Parameters
+###############################################################################
+
+parameter "param_processing_limit" do
+  type "number"
+  label "Hours Processing"
+  description "Number of hours between downloading and processing complete to be reported"
+  default 24
+end
+
+parameter "param_email" do
+  type "list"
+  label "Email addresses of the recipients you wish to notify"
+  description "A list of email addresses to notify"
+  min_length 1
+end
+
+###############################################################################
+# Authentication
+###############################################################################
+
+auth "auth_rs", type: "rightscale"
+
+###############################################################################
+# Datasources
+###############################################################################
+datasource "ds_bill_connects" do
+  request do
+    auth $auth_rs
+    verb "GET"
+    host rs_optima_host
+    path join(["/api/onboarding/orgs/", rs_org_id, "/bill_connects"])
+    header "Api-Version", "1.0"
+    header "User-Agent", "RS Policies"
+  end
+end
+
+datasource "ds_bill_connects_valid" do
+  request do
+    auth $auth_rs
+    verb "GET"
+    host rs_optima_host
+    path join(["/api/onboarding/orgs/", rs_org_id, "/bill_connects/validate"])
+    header "Api-Version", "1.0"
+    header "User-Agent", "RS Policies"
+  end
+end
+
+datasource "ds_bill_status_errors" do
+  request do
+    auth $auth_rs
+    verb "GET"
+    host rs_optima_host
+    path join(["/api/status/orgs/", rs_org_id, "/bill-status-errors"])
```

Comment:
currently this only returns CBI generic tranform errors.
So this is specifically to get errors that are not immediately available when uploading a CBI bill, like a missing column the integration expects.
This will not give any data for any of the other vendors, aws, azure, gcp, etc